package.json

```diff
@@ -31,7 +31,7 @@
     "run": "jpm run --addon-dir dist/ ${npm_config_jpm_runflags} --prefs jpm-prefs.json",
     "pretest": "npm run-script lint",
     "test": "cross-env NODE_ENV=test nyc mocha-webpack --opts ./mocha-webpack.opts --webpack-config ./webpack.config.test.js -r test/setup.js test",
-    "coveralls": "nyc report --reporter=text-lcov | coveralls",
+    "report-coverage": "npm run-script test && nyc report --reporter=lcov && codecov",
```

Comment:
I notice this changed to include `npm run-script test`. Is that necessary? Wouldn't that cause us to run the tests twice?

Reply:
Great catch. It should _not_ be necessary as test and coverage output will run by CI before this. Likely some residual from my local testing to make sure I actually had reports to work with. Thanks 🙇